src/minecraft/ResourcePackStack.h

```diff
@@ -8,7 +8,7 @@ class PackSettings;
 
 struct PackInstance {
 
-    char filler[0x7C];
+    char filler[0x8C];
```

Comment:
The real size of PackInstance is unknown, but must be greater than 0x7C.
No crash with 0x8C.
Fixed shutdown crash 1.12 server